dev/build-file-manifest

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
```

Comment:
Why not just add this logic into the promobot directly under a flag?

Also, what about tests?